safe_transaction_service/history/views.py

```diff
@@ -520,6 +520,7 @@ def swagger_safe_balance_schema(serializer_class):
 
 class SafeBalanceView(GenericAPIView):
     serializer_class = serializers.SafeBalanceResponseSerializer
+    pagination_class = pagination.ListPagination
```

Comment:
Why?

Reply:
I leaved it, don't make sense, removed in last commit.